acestep/models/common/guidance_registry_sampler_regression_test.py

```
@@ -0,0 +1,174 @@
+
+        # Registry path.
+        fn = get_guidance_fn("apg_classic")
+        state: dict = {}
+        reg_outputs = []
+        for pred_cond, pred_uncond, latents in sequence:
+            state["latents"] = latents
+            state["sigma"] = 0.5
+            state["step_role"] = "main"
+            reg_outputs.append(fn(pred_cond, pred_uncond, scale, state))
+
+        for ref, got in zip(reference_outputs, reg_outputs):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)